src/tests/JIT/opt/Vectorization/BufferMemmoveTailCall.il

```diff
@@ -53,7 +53,7 @@
     ldarg.0
     ldc.i4.s 64
     conv.i
-    tail. call void [System.Runtime]System.Buffer::Memmove(uint8&, uint8&, native uint)
+    tail. call void [System.Runtime]System.SpanHelpers::Memmove(uint8&, uint8&, native uint)
```

Comment:
This won't work SpanHelpers does not have forwarder in System.Runtime. The test would have to reference System.Private.CoreLib directly.

What is this test trying to test? Do we really need it? Tests that reference S.P.CoreLib directly are suspect.

Reply:
I don't think it adds a lot of value so I can remove it - it was testing Memmove being unrollable & tail-called.